x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_requirements_page/components/fleet_server_on_prem_instructions.tsx

```diff
@@ -196,41 +197,15 @@ export const FleetServerCommandStep = ({
           />
         </EuiText>
         <EuiSpacer size="l" />
-        <EuiButtonGroup
-          options={PLATFORM_OPTIONS}
-          idSelected={platform}
-          onChange={(id) => setPlatform(id as PLATFORM_TYPE)}
-          legend={i18n.translate('xpack.fleet.fleetServerSetup.platformSelectAriaLabel', {
-            defaultMessage: 'Platform',
-          })}
+        <PlatformSelector
+          linuxCommand={installCommand.linux}
+          macCommand={installCommand.mac}
+          windowsCommand={installCommand.windows}
+          linuxDebCommand={installCommand.deb}
+          linuxRpmCommand={installCommand.rpm}
+          troubleshootLink={docLinks.links.fleet.troubleshooting}
+          isK8s={false}
         />
-        <EuiSpacer size="s" />
```

Comment:
Changes in here look good to me. Will probably have a few conflicts but nothing super blocking. Thanks for making the edits here!